docs/src/content/docs/resources/themes.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Themes
+description: Style your docs with a community theme for Starlight
+tableOfContents: false
+sidebar:
+  order: 1
+head:
+  - tag: style
+    content: |
+      body { --sl-content-width: 66.75rem; }
+---
+
+import ThemeGrid from '~/components/theme-grid.astro';
+
+A theme is a Starlight plugin that changes the visual appearance of a site with [custom CSS](/guides/css-and-tailwind/), [component overrides](/guides/overriding-components/), or other new features.
+
+## Community Themes
+
+Install a theme built by the community to quickly customize the look and feel of your site.
+
+<ThemeGrid
+	labels={{
+		/** Accessible label for the theme toggle. */
+		toggle: 'Preview theme',
```

Comment:
Nit: a bit weird when using a screen reader imo, as it sounds like it's used for a control allowing you to preview the theme instead of selecting the theme of the previewed themes (yes, that's also a lof of "theme"s 😅 ). We use "Select theme" for the global theme picker, maybe we could use something similar here, e.g. "Select preview themes" or something similar?